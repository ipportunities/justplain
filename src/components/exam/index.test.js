import React from 'react'
import { shallow } from 'enzyme'
import Exam from './index'

describe('Exam', () => {
  it('should render correctly', () => {
    const component = shallow(<Exam />)
    expect(component).toMatchSnapshot()
  })
})
