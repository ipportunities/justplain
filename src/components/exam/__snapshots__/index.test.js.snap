// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exam should render correctly 1`] = `
<div>
  <Menu />
  <Switch>
    <Route
      path="/exam/edit/:id"
    >
      <ExamTypeEdit />
    </Route>
    <Route
      path="/exam/view/:examMomentId"
    >
      <ExamView />
    </Route>
  </Switch>
</div>
`;
